src/shogun/distance/Distance.h

```diff
@@ -120,6 +120,29 @@ class CDistance : public CSGObject
 			return distance(idx_a, idx_b);
 		}
 
+		/** NOT IMPLEMENTED
```

Comment:
Dont say not implemented.
Say this is an empty method that might be overloaded in a sub-class